addons/udes_stock/README.md

```diff
@@ -160,7 +163,7 @@ This is essentially a collection of products that are to be moved from one locat
 | _create_move              | Create and return move(s) for the given move_values. Should be used in conjunction with _prepare_move to obtain move_values                                                                                                                                                                                                                       |
 | get_empty_pickings        | Generate batches marked with u_is_empty in a recordset or picking type (if u_auto_unlink_pickings is on)                                                                                                                                                                                                                                          |
 | unlink_empty              | Deletes empty pickings generated by get_empty_pickings                                                                                                                                                                                                                                                                                            |
-
+| un_assign_users           | Un assigning the user(s) from the pickings with optional skip_users
```

Comment:
super nit:
 assigning the user(s) from the pickings ... ->  assigning the user(s) from the pickings from self ...

Reply:
Should that be the same as the docstring in the definition of method, didnt understand the nit sorry :(